### ISSUE TITLE
0.8.3: Custom validator check failed for prop "size"

### ISSUE THREAD

**Issue by dgattey** [buefy/buefy#1768]
### Overview of the problem

**Buefy** version: 0.8.3
**Vuejs** version: 2.5.4
**OS/Browser**: Chrome/Firefox/etc

### Description

Upgrading to 0.8.3 from 0.7 something brought us new errors on the Table component on every render. There's a `[Vue warn]: Invalid prop: custom validator check failed for prop "size".` that prints out, once per column, once per render. It happens whether or not there's data in the table, etc. We're using font-awesome icons.

### Steps to reproduce

Here's a small example that should give an example of how our table is structured:
```javascript
<b-table
      :data="data"
      :loading="loading.matches"
      :mobile-cards="false"
      paginated
      backend-pagination
      :total="total"
      :per-page="30"
      :current-page="page"
      @page-change="onPageChange"
      checkable
      :checked-rows.sync="$store.state.matches.new.selectedMatches"
      backend-sorting
      @sort="onSort"
      :default-sort="defaultSort"
      default-sort-direction="desc"
    >
        <!-- Status Column -->
        <b-table-column
          label="Interest"
          :sortable="true"
          field="sort_status_type"
          class="matches__table--status"
          :visible="matchesColumnIsVisible('interest')"
        >
          <!-- Internal components here... -->
          ...
        </b-table-column>
        ... [more columns here]
</b-table>
```

### Expected behavior

No error logs

### Actual behavior

Logging a warning for validation failing for prop "size" on table columns

**Comment by jtommy** [comment]
https://github.com/buefy/buefy/issues/1730 there is a workaround waiting in the next release

**Comment by service-paradis** [comment]
The warning is still there...
`vue.runtime.esm.js?2b0e:619 [Vue warn]: Invalid prop: custom validator check failed for prop "size".`

**Comment by service-paradis** [comment]
I think I understand what the problem is...
`vue-fontawesome` have a validator for the prop `size`:
`['lg', 'xs', 'sm', '1x', '2x', '3x', '4x', '5x', '6x', '7x', '8x', '9x', '10x'].indexOf(value) > -1`

The icons config file contains an empty value for `is-small`:
```
sizes: {
            'default': faIconPrefix + 'lg',
            'is-small': '',
            'is-medium': faIconPrefix + '2x',
            'is-large': faIconPrefix + '3x'
        },
```
which is invalid...

**Comment by service-paradis** [comment]
It is only a warning non harmful as I can see. But there is a workaround to prevent it of being displayed in the console:

```
Vue.use(Buefy, {
  defaultIconComponent: "vue-fontawesome",
  defaultIconPack: "fas",
  customIconPacks: {
    fas: {
      sizes: {
        default: "lg",
        "is-small": "1x",
        "is-medium": "2x",
        "is-large": "3x"
      }
    }
  }
});
```

**Comment by dgattey** [comment]
That workaround doesn’t seem to work for me

**Comment by Looooooka** [comment]
The workaround works for me.
I've been trying to figure out which component throws this out since I updated all my libraries a few days ago.
Turns out it was <b-table-column and...well this fixes it :)

**Comment by jtommy** [comment]
@Looooooka It has been fixed in the last release 0.8.5 anyway

**Comment by Looooooka** [comment]
Oh. Didn't even notice a new version was already out. I updated to 0.8.4 like a week ago I think. My bad.
Updating :)
